### PR TITLE
Add recipe for coursera-dl

### PR DESCRIPTION
**Pull request by hargup**
[Coursera-dl](https://github.com/coursera-dl/coursera) is a popular tool to download course content from coursera.org courses.

### CODE REVIEW

File: coursera-dl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - beautifulsoup4
```
Comment (ccordoba12) [inline coursera-dl/meta.yaml]: This package is called `beautiful-soup` in conda

File: coursera-dl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - mechanize
```
Comment (ccordoba12) [inline coursera-dl/meta.yaml]: `mechanize` doesn't exist in `conda-recipes`, so you'd need a recipe for it too.

File: coursera-dl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - argparse
```
Comment (ccordoba12) [inline coursera-dl/meta.yaml]: `argparse` is part of the standard library in Python 2.7, 3.3 and 3.4. So please remove this dependency.

Comment (groutr) [inline coursera-dl/meta.yaml]: or make it a dependency only for Python 2.6 with `- argparse     [py26]`

**Pull request closed without merging**